test/util/generator_helpers.rb

```diff
@@ -0,0 +1,150 @@
+require "yaml"
+
+require 'beaker-hostgenerator'
+require 'beaker-hostgenerator/data/vmpooler'
+require 'beaker-hostgenerator/data'
+
+module GeneratorTestHelpers
+  include BeakerHostGenerator::Data
+  include BeakerHostGenerator::Data::Vmpooler
+
+  def run_cli_with_options options = []
+    STDERR.reopen("stderr.txt", "w")
+    cli = BeakerHostGenerator::CLI.new(options)
+    yaml_string = cli.execute
+
+    hash = YAML.load(yaml_string)
+    return hash
```

Comment:
I would prefer to leave it in and deal with it in a more comprehensive rubocop-driven style refactor. I guess the implied return value of the last expression in the function is probably pretty familiar for you coming from clojure and all...I happen to prefer explicit over implicit for behaviors like this (coming from C/bash/python world) but will definitely be changing this in a later cleanup PR.